### PR TITLE
Add support to disable/enable roles on the appliance.

### PR DESCRIPTION
**Pull request by ghost**
This PR allows to enable or disable role when deploying the appliance. It takes two lists as variables: `miq_disabled_roles` and `miq_enabled_roles`. Both have the same default value `[]` that don't change the list of roles, so the appliance keeps the ManageIQ default roles.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by ghost** [comment]
@machacekondra, @mwperina could you please review this PR ?

File: tasks/manage_appliance_roles.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+- name: Add extra roles to list of active roles
+  set_fact:
+    miq_active_roles: "{{ miq_active_roles | union(miq_enabled_roles) }}"
+
+- name: Update list of active roles
+  uri:
+    url: https://{{ miq_ip_addr }}/api/servers/1/settings
+    user: "{{ miq_app_username | default('admin') }}"
+    password: "{{ miq_app_password | default('smartvm') }}"
```
Comment (machacekondra) [inline tasks/manage_appliance_roles.yml]: The '| default() ' isn't needed here. and for user.

File: README.md [hunk 2]
```diff
@@ -99,16 +99,18 @@ The item in `miq_vm_nics` list of can contain following attributes:
...
+| miq_app_username   | admin               | The username used to login to ManageIQ.                                    |
+| miq_app_password   | smartvm             | The password of user specific in username used to login to ManageIQ.       |
+| miq_username       | admin               | Alias of `miq_app_username` for backward compatibility.                    |
+| miq_password       | smartvm             | Alias of `miq_app_password` for backward compatibility.                    |
+| miq_db_username    | root                | The username to connect to the database.                                   |
+| miq_db_password    | `miq_app_password`  | The password of user specific in username used to connect to the database. |
+| miq_region         | 0                   | The ManageIQ region created in the database.                               |
+| miq_company        | My Company          | The company name of the appliance.                                         |
+| miq_disabled_roles | []                  | List of ManageIQ roles to disable on the appliance.                        |
```
Comment (machacekondra) [inline README.md]: maybe mentioning 'server roles' would be better? or somehow distinguish what roles are those

**Review by machacekondra — APPROVED** [review]

**Pull request merged**